hack/dockerfile/install/tomll.installer

```diff
@@ -4,6 +4,5 @@
 
 install_tomll() {
 	echo "Install go-toml version ${GOTOML_VERSION}"
-	# TODO remove GO111MODULE=on and change to 'go install -mod=mod ...' once we're at go 1.16+
-	GO111MODULE=on GOBIN="${PREFIX}" go get -v "github.com/pelletier/go-toml/cmd/tomll@${GOTOML_VERSION}"
+	GOBIN="${PREFIX}" GO111MODULE=on go install "github.com/pelletier/go-toml/cmd/tomll@${GOTOML_VERSION}"
```

Comment:
Wondering if we should just remove these scripts now, and inline this in the Dockerfile. WDYT @cpuguy83 @tianon ?

Reply:
You mean inline just the extra tools (like `tomll`), or all the `containerd`, etc too?  I'm +1 on the former but hesitant on the latter (since I think those scripts get used elsewhere too right?)

Reply:
Yes, correct, the containerd and runc scripts are still used in some other places, so we likely still need those. For the "CI utilities", I think we should be able to put them inside the dockerfile; they're just one-liners now, so having two scripts to run them is just unneeded overhead.